packages/ng/forms/text-input/text-input.component.html

```diff
@@ -15,13 +15,13 @@
 		}
 	</ng-template>
 
-	@if (prefix) {
+	@if (prefix()) {
```

Comment:
a good practise in strict mode is to use `as` in this case
```suggestion
	@if (prefix(); as prefix) {
```
and use `prefix` in the if block;

This way, `prefix` is not nil when used in subsequent calls